### ISSUE TITLE
Add deepTools to get bigWig coverage files

### ISSUE THREAD

**Issue by szilvajuhos** [nf-core/sarek#70]
bigWig is a compact format to get coverage information, and we can add it to IGV (https://github.com/igvteam/igv.js/wiki/Wig-Track) . With deepTools (https://deeptools.readthedocs.io/en/develop/index.html) we can easily calculate this file like

```bamCoverage -bs 10 -v -b Preprocessing/Recalibrated/sample.recal.bam --ignoreDuplicates --numberOfProcessors 48 --outFileName coverageDir/sample.coverage.bigWig``` 

It would be nice to have it among the results (and deepTools in the container, see conda install at: https://deeptools.readthedocs.io/en/develop/content/installation.html) . Maybe it can replace QualiMap and provide the same QC stuff.

**Comment by MaxUlysse** [comment]
Small issue, currently `Manta` is still in `Python 2.7` with no plan to move to `Python 3` yet: https://github.com/Illumina/manta/issues/180
This raises conflict when trying to resolve conda environment:

```bash
UnsatisfiableError: The following specifications were found to be in conflict:
  - deeptools=3.3.1
  - manta=1.6.0
```

**Comment by MaxUlysse** [comment]
I think I'll more into it next year, when we start looking more into modules

**Comment by MaxUlysse** [comment]
```
UnsatisfiableError: The following specifications were found to be in conflict:
  - deeptools=3.3.1 -> matplotlib[version='>=3.0.0'] -> matplotlib-base==3.0.2=py37h167e16e_1002
  - deeptools=3.3.1 -> matplotlib[version='>=3.0.0'] -> python_abi=[build=*_cp37m] -> pypy[version='<0a0']
  - tiddit=2.7.1
```
Still conflict, so I'll try again when I'm finished with DSL 2

**Comment by szilvajuhos** [comment]
OK, if we are moving towards 1-process-1-container, can we have this included somehow? Otherwise I can look how to generate bigWig with other tools.

**Comment by MaxUlysse** [comment]
Yes, definitively that's the plan, I'm just trying to release Sarek 2.6 first, and then it's full on DSL 2

**Comment by FriederikeHanssen** [comment]
@szilvajuhos after which step do you want this output? We have now implemented splitting the fastq files -> mapping splits in parallel -> duplicateMarking with implicit merging. Would deepTools after duplicate marking be ok? Otherwise we have to do the extra merging step again or you get a bunch of file for each read pair. From the documentation I understand the deep tools can only process one BAM file as input and not a list

**Comment by szilvajuhos** [comment]
Huh, being honest, I have no faintest idea. It seemed like a good idea at that time, but I do not think it is urgent to add if at all.

**Comment by FriederikeHanssen** [comment]
Ok, well I was about to smash some old issues now and this was on the list. In that case for simplicity I will add it in after duplicate marking, because that is when the other QC metrics are computed for now.

**Comment by FriederikeHanssen** [comment]
Added by PR #479 It was quick to add so here we go. let me know what you think :)
